### DISCUSSION TITLE
Click event on an element with wire:navigate

### DISCUSSION THREAD

**Discussion by joaopalopes24** [livewire/livewire#6341] [7. Bugs (archived)]
### Livewire version

v3.0.0-beta.8

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

I have a link on my page, and I want the link tag to be clicked when I click on a button. Without the wire:navigate in the link tag it works normally.

```
<button x-on:click="$refs.next.click()">
    Next
</button>

<a x-ref="next" href="/test">
    Título
</a>
```

If I put wire:navigate this example code stops working.

```
<button x-on:click="$refs.next.click()">
    Next
</button>

<a x-ref="next" href="/test" wire:navigate>
    Título
</a>
```

If I put a listener on the link tag, the click event is done.

```
<button x-on:click="$refs.next.click()">
    Next
</button>

<a x-on:click="console.log('Click')" x-ref="next" href="/test" wire:navigate>
    Título
</a>
```

Just to clarify, in the parent div I have the `x-data="{}"`.

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by gehrisandro** [comment]
I've opened a PR fixing this issue: https://github.com/livewire/livewire/pull/6995
